src/index.html

```diff
@@ -1,12 +1,325 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="Store Logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
+            <a class="nav__item nav__item--is--active is-active" href="#">Apple</a>
```

Comment:
you don't use class `is-active`.

also with BEM convention we should give name for modifier with one dash
```suggestion
            <a class="nav__item nav__item--is-active" href="#">Apple</a>
```

use this rule for future
`block-name__elem-name--mod-name--mod-val`